src/Asset/Controller/Export/ZipAssetController.php

```diff
@@ -55,31 +55,31 @@ public function __construct(
      * @throws ForbiddenException|NotFoundException|StreamResourceNotFoundException
      * @throws EnvironmentException|MaxFileSizeExceededException
      */
-    #[Route('/assets/zip/create', name: 'pimcore_studio_api_create_zip_asset', methods: ['POST'])]
+    #[Route('/assets/export/zip/asset', name: 'pimcore_studio_api_asset_export_zip_asset', methods: ['POST'])]
     #[IsGranted(UserPermissions::ASSETS->value)]
     #[Post(
-        path: self::PREFIX . '/assets/zip/create',
-        operationId: 'asset_create_zip',
-        description: 'asset_create_zip_description',
-        summary: 'asset_create_zip_summary',
+        path: self::PREFIX . '/assets/export/zip/asset',
+        operationId: 'asset_export_zip_asset',
+        description: 'asset_export_zip_asset_description',
+        summary: 'asset_export_zip_asset_summary',
         tags: [Tags::Assets->name]
     )]
     #[RequestBody(
-        content: new ScalarItemsJson('integer')
+        content: new ScalarItemsJson('integer', 'assets')
     )]
     #[CreatedResponse(
-        description: 'asset_create_zip_created_response',
+        description: 'asset_export_zip_created_response',
         content: new IdJson('ID of created jobRun', 'jobRunId')
     )]
     #[DefaultResponses([
         HttpResponseCodes::UNAUTHORIZED,
         HttpResponseCodes::NOT_FOUND,
```

Comment:
Missing HttpResponseCodes::MAX_FILE_SIZE_EXCEEDED? b/c of MaxFileSizeExceededException

Reply:
Good catch! I will add it